features/nanostack/mbed-mesh-api/mbed_lib.json

```diff
@@ -159,6 +159,10 @@
             "value_max": 255,
             "value": 255
         },
+        "wisun-device-type": {
+            "help": "Supported device operating modes: MESH_DEVICE_TYPE_WISUN_ROUTER, MESH_DEVICE_TYPE_WISUN_BORDER_ROUTER",
+            "value": "MESH_DEVICE_TYPE_WISUN_ROUTER"
```

Comment:
Would just MESH_DEVICE_TYPE_ROUTER be enough and sort of generic

Reply:
Those could be used also in 6LoWPAN

Reply:
We are using already two conventions for device type:

- 6Lowpan is already having `6lowpan-nd-device-type` with alues `NET_6LOWPAN_ROUTER` and `NET_6LOWPAN_HOST` are used. 

- thread is having `thread-device-type` with values `MESH_DEVICE_TYPE_THREAD_ROUTER`, `MESH_DEVICE_TYPE_THREAD_SLEEPY_END_DEVICE`, `MESH_DEVICE_TYPE_THREAD_MINIMAL_END_DEVICE` and `MESH_DEVICE_TYPE_THREAD_ROUTER`.

We just need to select if we follow 6LoWPAN or Thread convention here.